src/view_control.cpp

```diff
@@ -2140,6 +2141,15 @@ void View::DrawTempo(DeviceContext *dc, Tempo *tempo, Measure *measure, System *
         else
             params.m_x -= 2 * tempo->GetStart()->GetDrawingRadius(m_doc);
     }
+    else if (tempo->HasStartid()) {
+        LayerElement *parent = tempo->GetStart();
+        ClassIdComparison comp(ACCID);
+        ListOfObjects children;
+        parent->FindAllDescendantByComparison(&children, &comp);
```

Comment:
Actually - sorry of the late comment - you can use `Object::FindDescendantByType` since you need only the first one.

Reply:
Perhaps we should also consider the width of the glyph for alignment with the left edge?

Reply:
I'll open a new PR with improvements for the aforementioned things as soon as I've fixed them.